bluesky_queueserver/manager/manager.py

```diff
@@ -0,0 +1,655 @@
+import asyncio
+import json
+import aioredis
+import zmq
+import zmq.asyncio
+from multiprocessing import Process
+import threading
+import time as ttime
+import pprint
+import uuid
+
+from .worker import DB
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+"""
+#  The following plans that can be used to test the syste
+
+http POST 0.0.0.0:8080/add_to_queue plan:='{"name":"count", "args":[["det1", "det2"]]}'
+
+# This is the slowly running plan (convenient to test pausing)
+http POST 0.0.0.0:8080/add_to_queue plan:='{"name":"count", "args":[["det1", "det2"]],
+"kwargs":{"num":10, "delay":1}}'
+
+http POST 0.0.0.0:8080/add_to_queue plan:='{"name":"scan", "args":[["det1", "det2"], "motor", -1, 1, 10]}'
+"""
+
+
+class RunEngineManager(Process):
+    """
+    The class implementing Run Engine Worker thread.
+
+    Parameters
+    ----------
+    conn_watchdog: multiprocessing.Connection
+        One end of bidirectional (input/output) for communication to Watchdog process.
+    conn_worker: multiprocessing.Connection
+        One end of bidirectional (input/output) for communication to RE Worker process.
+    args, kwargs
+        `args` and `kwargs` of the `multiprocessing.Process`
+    """
+    def __init__(self, *args, conn_watchdog=None, conn_worker=None, **kwargs):
```

Comment:
```suggestion
    def __init__(self, *args, conn_watchdog, conn_worker, **kwargs):
```

This will make them required kwargs (py36 feature iirc).